packages/core/src/alert/alert-actions.element.scss

```diff
@@ -2,6 +2,8 @@
 @import './../styles/mixins/utils';
 
 :host {
+  --action-hover-text-color: #{$cds-token-color-neutral-1000};
+  --action-text-color: #{$cds-token-color-neutral-700};
```

Comment:
Moved vars to initialize `:host` as they are required for the polyfill and the next version of the API generator